ros-controls/ros2_controllers
# [JTC] Adding tests for acceleration command interface

I found todos concerning acceleration tests:
- Adds checks for acceleration command interfaces to  
  - `test_jumbled_joint_order` and
  - `test_partial_joint_list`
- Use variable for `test_jumbled_joint_order` goal. 
- Change logic of `is_same_sign` to `is_same_sign_or_zero`

Minor changes
- use `std` namespace for `copysign`

If someone is thinking about the test trajectory points and the expected outputs of `test_jumbled_joint_order`
![image](https://github.com/ros-controls/ros2_controllers/assets/3367244/791066a5-220e-4431-ba95-c359a3a6351a)
